sklearn/svm/base.py

```diff
@@ -541,6 +541,15 @@ def decision_function(self, X):
             in the model.
             If decision_function_shape='ovr', the shape is (n_samples,
             n_classes)
+
+        Notes
+        ------
+        The function values are proportional to the distance of the samples X
```

Comment:
This is only relevant to decision_function_shape='ovo' isn't it? For ovr it does something more complicated

Reply:
Good point. As far as I understand [the code](https://github.com/scikit-learn/scikit-learn/blob/55bf5d93e5674f13a1134d93a11fd0cd11aabcd1/sklearn/utils/multiclass.py#L402), the decision values are somehow weighted up together and then scaled to a defined range. It does probably not make any sense to derive to the real distances based on the confidence values. So, yes, the note is only relevant to 'ovo' decision functions.

I would suggest changing the intro of the note to
`If decision_function_shape=’ovo’, the function values are proportional`...
is that ok?

Reply:
Yes, just qualifying the statement like that s fine.

Reply:
Ok, done :-)